res/WELCOME.md

```diff
@@ -29,7 +29,7 @@ An alternative is the channel [`#git-devel`](https://webchat.freenode.net/#git-d
 
 Once on the list of permitted usernames, you can contribute the patches to the Git mailing list by adding a PR comment `/submit`.
 
-After you submit, GitGitGadget will respond with another comment that contains the link to the cover letter mail in the Git mailing list archive. Please make sure to monitor the discussion in that thread and to address comments and suggestions.
+After you submit, GitGitGadget will respond with another comment that contains the link to the cover letter mail in the Git mailing list archive. Please make sure to monitor the discussion in that thread and to address comments and suggestions. To iterate on your change, push to the same branch or modify your pull request title or description, and comment `/submit` again.
```

Comment:
Maybe "(force-)push" instead of "push"?

Also, I don't think that GitGitGadget lets you submit a new iteration with only a changed PR description; it requires a change to at least one of the commits.

Finally, how about suggesting to summarize the changes in the cover letter, e.g. like this:

```
Changes since v1:
- Fixed a typo in the commit message (found by ...)
- Added a code comment to ... as suggested by ...
...
```